src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 101;
+    private Ball lotteryBall;
+
+    public Ball getRandomBall() {
+        lotteryBall = new Ball();
+        lotteryBall.setColor(new ColorSupplier().getRandomColor());
+        lotteryBall.setNumber(new Random().nextInt(MAX_BALL_NUMBER));
```

Comment:
Think about which variables should be local in the method and which should be class-level. You need to create a new Ball every time someone calls this method, so there is no point in storing a Ball on a class level because you cant reuse it in further invocations of this method. The opposite thing I can say about ColorSupplier and Random. You don't need to recreate them on each call of getRandomBall(), you can store them on the class level and reuse.